ui/main/src/app/services/card.service.ts

```diff
@@ -108,10 +108,14 @@ export class CardService {
 
     private getCardSubscription(): Observable<CardOperation> {
         // security header needed here as SSE request are not intercepted by our header interceptor
+        let securityHeader;
+        if (!this.authService.isAuthModeNone) {
```

Comment:
in fact it is : ` if (!this.authService.isAuthModeNone())`  otherwise it does not work as it tests the existence of the function and not the return value

Reply:
Yes, fixed it in the new commit. Thanks for spotting that!